lib/kino/process.ex

```diff
@@ -137,6 +139,212 @@ defmodule Kino.Process do
     """)
   end
 
+  @doc """
+  Generate a sequence diagram of process messages.
```

Comment:
Now that you added `:all` and pids, it is important to clarify that the trace starts from the the current process.

```suggestion
  Generate a sequence diagram of process messages starting from `self()`.
```